packages/react/src/primitives/Menu/Menu.tsx

```diff
@@ -33,7 +33,7 @@ const MenuPrimitive: Primitive<MenuProps, 'div'> = (
     <DropdownMenuTrigger asChild={true}>
       {trigger ?? (
         <MenuButton
-          role="button"
```

Comment:
not sure why we need to add it for a button, so removed it

Reply:
Nope, it doesn't cause any a11y error because it is a button. It would cause error if it's a `<div>` or something but uses as a button

Reply:
I just found that our `<MenuButton>` is `button`, but `<Button>` is `div` 😱  

So we can remove `role` here, but we need to keep `role` for `<Button>`

Reply:
Tested it and the menu button still works correctly. Proceed 👍